modules/ROOT/examples/configuration/sync-gateway-config.json

```diff
@@ -110,3 +110,371 @@
+    }
+    `
+// other config as required
+  }
+]
+----
+// end::icr-simple-own-winner-conflict-resolver[]
+
+// tag::icr-simple-merge-conflict-resolver[]
+
+[source, json]
+----
+"replication": [
+  {
+  "replication_id": "replication1",
+  // other config as required
+  "conflict_resolution_type": "custom",
+  "custom_conflict_resolver": `
+    function(local, remote) {
+      if (doc.type == "a-doctype-1") {
+          // Invoke the built in resolver logic for remoteWins
+          Return RemoteWins(doc,oldDoc);
+        }
+        else {
+          // Invoke the built in default resolver logic
+          Return UseDefault(doc,oldDoc);
+        }
+    }
+    `
+// other config as required
+  }
+]
+----
+// end::icr-simple-merge-conflict-resolver[]
+
+// tag::NEW-icr-sample-sync-gateway-config[]
+[source, json]
+----
+{
+  "interface":":5984",
+  "adminInterface":":5985",
+  "log": ["*"],
+  "logging": {
+    "log_file_path": "/var/tmp/sglogs",
+    "console": {
+      "log_level": "debug",
+      "log_keys": ["*"]
+    },
+    "error": {
+      "enabled": true,
+      "rotation": {
+        "max_size": 20,
+        "max_age": 180
+      }
+    },
+    "warn": {
+      "enabled": true,
+      "rotation": {
+        "max_size": 20,
+        "max_age": 90
+      }
+    },
+    "info": {
+      "enabled": false
+    },
+    "debug": {
+      "enabled": false
+    }
+  },
+  "databases": {
+       "db-local": {
+            "import_docs": true,
+            "bucket":"db-local",
+            "server": "couchbase://cb-server",
+            "enable_shared_bucket_access":true,
+            "delta_sync": {
+              "enabled": true
+            },
+            "import_filter": `
+              function(doc) {
+                return true;
+              }
+              `,
+            "username": "admin",
+            "password": "password",
+            "users":{
+                "admin": {"password": "password", "admin_channels": ["*"]},
+                "user1": {"password": "password", "admin_channels":["channel.user1"]}
+            },
+           "num_index_replicas":0,
+           "sgreplicate_enabled":false,
+           "replications":{
+               "pushandpull-with-target-continuous-1" : {
+                   "direction": "pushAndPull",
+                   "conflict_resolution_type":"custom",
+                    "custom_conflict_resolver":`
+                                function(conflict) {
+                                    var remoteDoc = conflict.RemoteDocument;
+                                     console.log("full remoteDoc doc: "+JSON.stringify(remoteDoc));
+                                    var localDoc = conflict.LocalDocument;
+                                    console.log("full remoteDoc doc: "+JSON.stringify(localDoc));
+                                    var mergedDoc = {...localDoc,...remoteDoc};
+                                    console.log("full remoteDoc doc: "+JSON.stringify(mergedDoc));
```

Comment:
It looks like the last line of logging should be 'full mergedDoc doc'.

Also, mergedDoc._rev will need to be set to "" when taking this approach.